src/services/telegram.ts

```diff
@@ -7,7 +7,7 @@ import { quoteToImage } from "services/canvasMaker";
 const Bot = new Telegraf(process.env.TELEGRAM_TOKEN);
 
 Bot.start(replyQuote);
-Bot.hears(/kanye|քանյե/i, replyQuote);
+Bot.hears(/kanye|քանյե|ye|յէ|goat|🐐/i, replyQuote);
```

Comment:
this will make the bot reply to texts such as: `yerevan, յէ բոոի` etc.
for this to work you'd need to add standalone messages for each keyword instead of adding them to the current regexp.
or modify the regexp to match exact keywords instead of all the strings that include the keyword.

Reply:
I think it should react at every possibility of the name mentioned, but fine, I'll do the changes

Reply:
@rasjonell please check again

Reply:
I think you should add `\b` to both ends of the keyword like so:
`/kanye|քանյե|goat|🐐|\bye\b|\bյե\b/i`

_with `\b` only in the beggining_
![](https://imgur.com/lZMpAOs.png)

_`\b` on both ends_
![](https://imgur.com/7pZiAGQ.png)